packages/ui/components/TopBar.vue

```diff
@@ -17,7 +17,9 @@ const { toggle } = useNavigationDrawer();
 
 <template>
   <v-app-bar elevation="0" color="black">
-    <v-btn class="d-md-none" variant="text" icon="mdi-menu" @click.stop="toggle" />
+    <v-btn class="d-md-none" variant="text" icon @click.stop="toggle">
```

Comment:
this should be `type="button"` removing the `.stop` modifier

Reply:
Now I understand the difference :)